src/test/resources/org/broadinstitute/barclay/help/templates/BashTestDoclet/bash-completion.template.ftl

```diff
@@ -0,0 +1,343 @@
+
+####################
+# Tab completion file to allow for easy use of this tool with the command-line (bash/zsh)
+####################
+
+<#include "bash-completion.macros.ftl"/>
+
+####################################################################################################
+
+# High-level caller/dispatch script information:
```

Comment:
I prefer to have in Barclay something that is not gatk specific, for being able to use the default template in other toolkits as a start point without exploring the templates...